src/kafka-util/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "kafka-util"
+version = "0.1.0"
+authors = ["Brennan Vincent <brennan@materialize.io>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+clap = "2.33"
```

Comment:
I'd really like to avoid introducing a _third_ option parsing library into our codebase. (We already use structopt and getopts.) You mind if I rewrite this to use getopts after it lands?

Reply:
I agree that for code hygiene we should limit this, but in terms of build times StructOpt depends on clap so we shouldn't be dramatically increasing build times by doing this.